src/screens/login/index.tsx

```diff
@@ -8,58 +8,108 @@ import { login, registration } from '../../api/userAPI';
 import Background from '../../components/background/Background';
 import Title from '../../components/title/Title';
 
+import { ERROR_MESSAGE } from '../../../errors/errors';
 import { Icons } from '../../assets/Icons';
 import { Colors } from '../../constants/colors';
 import { TemplateType } from '../../constants/templateType';
 import { StackParams, UserType } from '../../types/types';
 
-import { ButtonText, LoginFrame, LoginInput, SignInButton, SignUpButton } from './styles';
+import { ButtonText, ErrorLogin, ErrorWindow,LoginFrame, LoginInput, SignInButton, SignUpButton, TextError } from './styles';
 
 const Login: FC = () => {
   const navigation = useNavigation<NativeStackNavigationProp<StackParams>>();
   const [email, setEmail] = useState<string | undefined>();
   const [password, setPassword] =useState<string | undefined>();
   const [user, setUser] = useState<UserType | undefined>();
+  const [isError, setIsError] = useState<boolean | undefined>(false );
+  const [errorMessage, setErrorMessage] = useState<string | undefined>();
 
   const signIn = async (): Promise<void> => {
     const userData = await login(email, password);
```

Comment:
wrap this in try catch and write something like:
```
setErrorMessage(null);
try {
  const userData = await login(email, password);
  setUser(userData as UserType);
  navigation...
} catch (e) {
   setErrorMessage(ERROR_MESSAGE[e]);
}
```
get rid of isError, you already have a state which is responsible for errors (errorMessage)